src/axom/primal/tests/primal_triangle.cpp

```diff
@@ -151,11 +147,73 @@ TEST(primal_triangle, triangle_physical_to_bary)
   }
 }
 
+//------------------------------------------------------------------------------
+TEST(primal_triangle, triangle_unnormalized_bary)
+{
+  constexpr int DIM = 3;
+  constexpr double EPS = 1e-12;
+  using CoordType = double;
+  using QPoint = primal::Point<CoordType, DIM>;
+  using QTri = primal::Triangle<CoordType, DIM>;
+
+  QPoint pt[3] = {QPoint {1, 0, 0},  //
```

Comment:
delete extra // if no comment

Reply:
@dtaller you don't find that comment useful? 😄

Reply:
That one is actually to get the formatter (clang-format) to respect my line breaks. 
Might be less important on this line now that we can initialize points using the (curly brace) initializer lists.

Reply:
AND I support getting the formatter to respect your line breaks.  : )

Reply:
Ok, that makes sense, no worries. You don't need to fix clang formatter in this pull request.